src/main/resources/application.yaml

```diff
@@ -123,4 +123,4 @@ azure:
     instrumentation-key: ${APPINSIGHTS_INSTRUMENTATIONKEY:dummy}
 
 features:
-  federated-s-s-o: true
\ No newline at end of file
+  federated-s-s-o: ${federated.sso:false}
```

Comment:
not sure, should we enable this by default?
```suggestion
  federated-s-s-o: ${federated.sso:true}
```

Reply:
we only want it off in prod right?

Reply:
That's the thing @nikola-naydenov-hmcts , I don't know :) Should I accept the suggestion?